scraper/maiia/maiia.py

```diff
@@ -1,117 +1,120 @@
 import json
 import logging
 
-import requests
+import httpx
 from datetime import datetime, timedelta
 from dateutil.parser import isoparse
 from pathlib import Path
-from bs4 import BeautifulSoup
-from scraper.profiler import Profiling
+from urllib.parse import quote
 
+from scraper.profiler import Profiling
+from scraper.pattern.center_info import get_vaccine_name
 from scraper.pattern.scraper_request import ScraperRequest
+from scraper.maiia.maiia_utils import get_paged, MAIIA_LIMIT
+
+timeout = httpx.Timeout(30.0, connect=30.0)
+DEFAULT_CLIENT = httpx.Client(timeout=timeout)
+logger = logging.getLogger('scraper')
 
-BASE_AVAILIBILITY_URL = "https://www.maiia.com/api/pat-public/availability-closests"
+MAIIA_URL = 'https://www.maiia.com'
 MAIIA_DAY_LIMIT = 50
-MAIIA_LIMIT = 10000
 
-session = requests.Session()
-logger = logging.getLogger('scraper')
 
+def parse_slots(slots: list) -> datetime:
```

Comment:
Si on peut renvoyer un objet `datetime` ou `None`, il faut écrire : 
```suggestion
def parse_slots(slots: list) -> Optional[datetime]:
```
(ne pas oublier `from typing import Optional`)